lib/slide_hub/io.rb

```diff
@@ -0,0 +1,9 @@
+class IO
+  def head(n)
+    lazy.first(n)
+  end
+
+  def tail(n)
```

Comment:
__[Reek]__

[UncommunicativeParameterName] IO#tail has the parameter name 'n'


<!-- Comment by [SideCI](https://sideci.com) -->